tests/sample_size/test_sample_size_calculator.py

```diff
@@ -150,13 +83,15 @@ def test_get_single_sample_size_ttest(self, mock_solve_power):
 
     @patch("sample_size.sample_size_calculator.SampleSizeCalculator._get_single_sample_size")
     def test_get_overall_sample_size_bool(self, mock_get_single_sample_size):
```

Comment:
Can you add a comment (and a JIRA ticket if you feel like it) to eventually parameterize these `get_overall` and  `register` test cases?